### PR TITLE
add direct vertical aiming (from Crispy Doom)

### LINKED ISSUE #745: Vertical aiming
Have a habit playing with vertical aiming, and that's the feature i am missing in woof.
Tried to port it back from nugget doom but failed miserably.
Meanwhile it is in crispy.
Please, pretty please.

### PR DESCRIPTION
**Pull request by fabiangreffrath**
Fix #745

### CODE REVIEW

File: src/p_mobj.h [hunk 1]
```diff
@@ -416,6 +416,9 @@ extern int iquetail;
 // [FG] colored blood and gibs
 extern boolean colored_blood;
 
+#define PLAYER_SLOPE(a)	((((a)->lookdir / MLOOKUNIT) << FRACBITS) / 173)
```
Comment (MrAlaux) [inline src/p_mobj.h:419]: I've been wondering: where does the `173` come from? On a related note, does this match the vertical aiming range of autoaim?

Comment (fabiangreffrath) [reply src/p_mobj.h:419]: Short answer, from Heretic: 😁 
https://github.com/chocolate-doom/chocolate-doom/blob/e96d2c2c1bf33b3367cf40d234563d7c5465b614/src/heretic/p_pspr.c#L818

Longer answer, honestly I don't know, but I'd love to de-obfuscate it.

Comment (rfomin) [reply src/p_mobj.h:419]: Maybe it's angle to fixed conversion? https://github.com/team-eternity/eternity/blob/cdac0335080c09e3eb38ed560de0de0f542cc3c7/source/tables.h#L106

Comment (rfomin) [reply src/p_mobj.h:419]: Also: https://github.com/team-eternity/eternity/blob/cdac0335080c09e3eb38ed560de0de0f542cc3c7/source/tables.h#L71-L73

**Pull request merged**